source/blender/riistudio_blender.py

```diff
@@ -1601,38 +2796,45 @@ def register_mat():
 			# ('logic', 'Do not blend', ''),
 			('subtract', 'Subtract from Frame Buffer', ''),
 		],
-		default="none"
+		default="none",
+		update=dum_pe_update
 	)
 	bpy.types.Material.jres_pe_blend_source = EnumProperty(
 		name="",
 		items = BLEND_MODE_FACTORS,
-		default="dst_a"
+		default="src_a",
+		update=dum_pe_update
 	)
 	bpy.types.Material.jres_pe_blend_dest = EnumProperty(
 		name="",
 		items = BLEND_MODE_FACTORS_2,
-		default="inv_dst_a"
+		default="inv_src_a",
+		update=dum_pe_update
 	)
 	bpy.types.Material.jres_pe_dst_alpha = IntProperty(
 		name="Value",
 		min=0,
 		max=255,
 		default=0,
+		update=dum_pe_update
 	)
 	bpy.types.Material.jres_pe_dst_alpha_enabled = BoolProperty(
 		name="Enabled",
 		default=False,
+		update=dum_pe_update
 	)
 
 	# Lighting
 	bpy.types.Material.jres_lightset_index = IntProperty(
 		name="Lightset Index",
-		default=-1
+		default=-1,
+		update=dum_scene_update
 	)
 	# Fog
 	bpy.types.Material.jres_fog_index = IntProperty(
 		name="Fog Index",
-		default=0
+		default=0,
+		update=dum_scene_update
```

Comment:
Generally it's good practice to always include a trailing comma to avoid diff noise like this. I was definitely bad about this in the past, but we should probably do that on everything in this file moving forward.
```diff
- update=dum_scene_update
+ update=dum_scene_update,
```